framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -285,104 +283,101 @@ protected boolean areEqualInHierarchy(
   }
 
   /**
-   * A declared type is considered a supertype of another declared type only if all of the type
-   * arguments of the declared type "contain" the corresponding type arguments of the subtype.
-   * Containment is described in <a
+   * Returns true if {@code outside} contains {@code inside}, that is, if the set of types denoted
+   * by {@code outside} is a superset of or equal to the set of types denoted by {@code inside}.
+   *
+   * <p>Containment is described in <a
    * href="https://docs.oracle.com/javase/specs/jls/se11/html/jls-4.html#jls-4.5.1">JLS section
-   * 4.5.1 "Type Arguments of Parameterized Types"</a>.
+   * 4.5.1 "Type Arguments of Parameterized Types"</a>. A declared type is considered a supertype of
```

Comment:
This second sentence is a reference to a *different* section of the JLS.  Please mention that: "In JLS section XX, ...".